process.c

```diff
@@ -4679,56 +4679,132 @@ rb_spawn(int argc, const VALUE *argv)
+ *
+ *  The first required argument is one of the following:
+ *
+ *  - +command_line+ if it is a string,
+ *    and if it begins with a shell reserved word or special built-in,
+ *    or if it contains one or more metacharacters.
+ *  - +exe_path+ otherwise.
+ *
+ *  <b>Argument +command_line+</b>
+ *
+ *  \String argument +command_line+ is a command line to be passed to a shell;
+ *  it must begin with a shell reserved word, begin with a special built-in,
+ *  or contain meta characters:
+ *
+ *    system('echo')                                  # => true  # Built-in.
+ *    system('if true; then echo "Foo"; fi')          # => true  # Shell reserved word.
+ *    system('date > /tmp/date.tmp')                  # => true  # Contains meta character.
+ *    system('date > /nop/date.tmp')                  # => false
+ *    system('date > /nop/date.tmp', exception: true) # Raises RuntimeError.
+ *
+ *  Assigns the command's error status to <tt>$?</tt>:
+ *
+ *    system('echo')                             # => true  # Built-in.
+ *    $?                                         # => #<Process::Status: pid 640610 exit 0>
+ *    system('date > /nop/date.tmp')             # => false
+ *    $?                                         # => #<Process::Status: pid 640742 exit 2>
+ *
+ *  The command line may also contain arguments and options for the command:
+ *
+ *    system('echo "Foo"') # => true
+ *
+ *  Output:
+ *
+ *    Foo
+ *
+ *  On a Unix-like system, the shell is <tt>/bin/sh</tt>;
+ *  otherwise the shell is determined by environment variable
+ *  <tt>ENV['RUBYSHELL']</tt>, if defined, or <tt>ENV['COMSPEC']</tt> otherwise.
+ *
+ *  Except for the +COMSPEC+ case,
+ *  the entire string +command_line+ is passed as an argument
+ *  to {shell option -c}[https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/sh.html].
  *
- *     config.h main.rb
- *     *
+ *  The shell performs normal shell expansion on the command line:
+ *
+ *    system('echo C*') # => true
  *
- *  Error handling:
+ *  Output:
+ *
+ *    CONTRIBUTING.md COPYING COPYING.ja
  *
- *     system("cat nonexistent.txt")
- *     # => false
- *     system("catt nonexistent.txt")
- *     # => nil
+ *  Raises an exception if the new process could not execute.
  *
- *     system("cat nonexistent.txt", exception: true)
- *     # RuntimeError (Command failed with exit 1: cat)
- *     system("catt nonexistent.txt", exception: true)
- *     # Errno::ENOENT (No such file or directory - catt)
+ *  <b>Argument +exe_path+</b>
+ *
+ *  Argument +exe_path+ is one of the following:
+ *
+ *  - The string path to an executable to be called.
+ *  - A 2-element array containing the path to an executable
+ *    and the string to be used as the name of the executing process.
+ *
+ *  Example:
+ *
+ *    system('/usr/bin/date') # => true # Path to date on Unix-style system.
+ *    system('foo')           # => nil   # Command failed.
```

Comment:
Nit: alignment is off from the line above.

```suggestion
 *    system('foo')           # => nil  # Command failed.
```